crates/red_knot_python_semantic/resources/mdtest/type_properties/is_disjoint_from.md

```diff
@@ -353,3 +363,27 @@ class UsesMeta2(metaclass=Meta2): ...
 
 static_assert(is_disjoint_from(type[UsesMeta1], type[UsesMeta2]))
 ```
+
+## Callables
+
+No two callable types are disjoint because there exists a callable type
+`(*args: object, **kwargs: object) -> Never` that is a subtype of all fully static callable types.
```

Comment:
Similar to Alex's suggestion, but I think it's important to clarify that it's a non-empty subtype, because even disjoint types have the common subtype `Never`. And personally I think once we clarify that, it's sufficient without adding an entirely new sentence: if two types have a non-empty common subtype, then obviously they share some inhabitants. But I don't feel strongly, we can add another sentence driving that point home.

```suggestion
No two callable types are disjoint because there exists a non-empty callable type
`(*args: object, **kwargs: object) -> Never` that is a subtype of all fully static callable types.
```